src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamespaceSymbol.cs

```diff
@@ -106,27 +106,21 @@ public sealed override ImmutableArray<Symbol> GetMembers(ReadOnlyMemory<char> na
                 if (lazyTypes.TryGetValue(name, out t))
                 {
                     // TODO - Eliminate the copy by storing all members and type members instead of non-type and type members?
-                    var result = StaticCast<Symbol>.From(t).Add(ns);
-                    _lastGetMembersResult = (name, result);
-                    return result;
+                    return _lastGetMembersResult = StaticCast<Symbol>.From(t).Add(ns);
                 }
                 else
                 {
-                    var result = ImmutableArray.Create<Symbol>(ns);
-                    _lastGetMembersResult = (name, result);
-                    return result;
+                    return _lastGetMembersResult = ImmutableArray.Create<Symbol>(ns);
```

Comment:
💡 From my measurements, this is the only line we need to be caching. Caching the other lines seems like it would make the cache less beneficial.

Reply:
i would far prefer this to be two statements.